apps/api/src/app/workflows/dto/workflow-response.dto.ts

```diff
@@ -115,4 +116,13 @@ export class WorkflowResponse {
   @ApiPropertyOptional()
   @IsOptional()
   data?: NotificationTemplateCustomData;
+  activeIntegrationStatus?: ActiveIntegrationsChannelsStatus;
+
+  @ApiProperty()
+  isBlueprint: boolean;
```

Comment:
Should we really expose these in the swagger json? right now this is only used internally for use?

Reply:
Agree, we shouldn't expose it, let me remove it